README.md

```diff
@@ -81,128 +159,79 @@ LocaleConfig.locales['fr'] = {
     'Novembre',
     'Décembre'
   ],
+  monthNames: ['Janvier', 'Février', 'Mars', 'Avril', 'Mai', 'Juin', 'Juillet', 'Août', 'Septembre', 'Octobre', 'Novembre', 'Décembre'],
   monthNamesShort: ['Janv.', 'Févr.', 'Mars', 'Avril', 'Mai', 'Juin', 'Juil.', 'Août', 'Sept.', 'Oct.', 'Nov.', 'Déc.'],
   dayNames: ['Dimanche', 'Lundi', 'Mardi', 'Mercredi', 'Jeudi', 'Vendredi', 'Samedi'],
   dayNamesShort: ['Dim.', 'Lun.', 'Mar.', 'Mer.', 'Jeu.', 'Ven.', 'Sam.'],
   today: "Aujourd'hui"
 };
-LocaleConfig.defaultLocale = 'fr';
-```
 
-### Calendar
+LocaleConfig.defaultLocale = 'fr';
 
-<kbd>
-  <img src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/assets/calendar.gif?raw=true">
-</kbd>
+const App = () => {
+  const [selected, setSelected] = useState('');
+
+  return (
+    <Calendar
+      onDayPress={(day) => {
+        setSelected(day.dateString);
+      }}
+      markedDates={{
+        [selected]: { selected: true, disableTouchEvent: true, selectedDotColor: 'orange' },
+      }}
+    />
+  );
+};
 
-#### Basic parameters
+export default App;
+```
 
+### Displaying the calendar in a custom style:
```

Comment:
More like adding a global theme to the calendar (we have the custom style example above...)